fonduer/supervision/lf_helpers/__init__.py

```diff
@@ -0,0 +1,147 @@
+import logging
+
+from fonduer.supervision.utils.structural import (
+    common_ancestor,
+    get_ancestor_class_names,
+    get_ancestor_id_names,
+    get_ancestor_tag_names,
+    get_attributes,
+    get_next_sibling_tags,
+    get_parent_tag,
+    get_prev_sibling_tags,
+    get_tag,
+    lowest_common_ancestor_depth,
+)
+from fonduer.supervision.utils.tabular import (
+    same_document,
+    same_table,
+    same_row,
+    same_col,
+    is_tabular_aligned,
+    same_cell,
+    same_sentence,
+    get_max_col_num,
+    get_min_col_num,
+    get_sentence_ngrams,
+    get_neighbor_sentence_ngrams,
+    get_cell_ngrams,
+    get_neighbor_cell_ngrams,
+    get_row_ngrams,
+    get_col_ngrams,
+    get_aligned_ngrams,
+    get_head_ngrams,
+)
+from fonduer.supervision.utils.textual import (
+    get_between_ngrams,
+    get_left_ngrams,
+    get_right_ngrams,
+)
+from fonduer.supervision.utils.visual import (
+    get_page,
+    is_horz_aligned,
+    is_vert_aligned,
+    is_vert_aligned_left,
+    is_vert_aligned_right,
+    is_vert_aligned_center,
+    same_page,
+    get_horz_ngrams,
+    get_vert_ngrams,
+    get_page_vert_percentile,
+    get_page_horz_percentile,
+    get_visual_aligned_lemmas,
+    get_aligned_lemmas,
+)
+
+
+def is_superset(a, b):
```

Comment:
This function and overlap() seemed too general to be in tabular.py, so I moved them here.

Reply:
👍